src/lib/styles/styles.scss

```diff
@@ -320,3 +320,173 @@ terra-alert-panel, *
+                    }
+                }
+            }
+        }
+    
+        &.mat-form-field-disabled
+        {
+            .mat-form-field-wrapper
+            {
+                .mat-form-field-underline
+                {
+                    border-bottom: inherit;
+                }
+    
+                .mat-form-field-flex
+                {
+                    .mat-form-field-infix
+                    {
+                        .mat-form-field-label-wrapper
+                        {
+                            .mat-form-field-label
+                            {
+                                mat-label
+                                {
+                                    color: var(--color-structure-6);
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+        
+        .mat-form-field-wrapper
+        {
+            padding-bottom: 0;
+            
+            .mat-form-field-flex
+            {
+                .mat-form-field-infix
+                {
+                    border-top: 1em solid transparent;
+                    padding: calc(var(--space-sm) * 1.5) 0 calc(var(--space-sm) * 1.5) 0;
+                    
+                    mat-select
+                    {
+                        .mat-select-trigger
+                        {
+                            .mat-select-value
+                            {
+                                .mat-select-value-text
+                                {
+                                    color: var(--color-structure-12);
+                                    font-weight: var(--font-weight-2);
+                                }
+                            }
+                        }
+                    }
+    
+                    .mat-form-field-label-wrapper
+                    {
+                        .mat-form-field-label
+                        {
+                            mat-label
+                            {
+                                color: var(--color-structure-7);
+                            }
+                        }
+                    }
+                }
+            }
+            
+            .mat-form-field-underline
+            {
+                border-bottom: var(--input-wrapper-border, 1px solid var(--border-color));
+                bottom: 0em;
+                
+                .mat-form-field-ripple
+                {
+                    background-color: var(--color-active);
+                }
+            }
+        }
+    }
+}
+
+.mat-select-panel
+{
+    &.mat-primary
+    {
+        mat-option
+        {
+            &.mat-selected
+            {
+                &:not(.mat-option-disabled)
+                {
+                    color: var(--color-structure-0);
+                    background-color: var(--color-active);
+                }
+            }
+        }
+    }
+}
```

Comment:
no new line
&
Wollten wir nicht eine extra Datei für jede neue Komponente erstellen anstatt sie in styles reinzumachen? Allein das sind schon fast 200 Zeilen.